cvat/nginx.conf

```diff
@@ -0,0 +1,80 @@
+worker_processes auto;
+pid /tmp/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+  worker_connections 512;
+}
+
+http {
+
+  ##
+  # Basic Settings
+  ##
+
+  sendfile on;
+  tcp_nopush on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  ##
+  # Logging Settings
+  ##
+
+  access_log /home/django/logs/nginx_access.log;
+  error_log /home/django/logs/nginx_error.log;
+
+  ##
+  # CVAT Settings
+  ##
+
+  server {
+    listen 8080;
+    # previously used value
+    client_max_body_size 1G;
+
+    server_name _;
+
+    location /static/ {
+      gzip on;
+      gzip_comp_level 6;
+      gzip_http_version 1.1;
+      gzip_types
+          text/html
+          application/javascript
+          application/x-javascript
+          text/javascript
+          application/wasm
+          image/x-icon;
+
+      root /home/django;
```

Comment:
I'm not familiar with nginx configuration, but doesn't that mean that _everything_ under `/root/django` will be served, including source code?

Reply:
Shouldn't be true because 'location' is not '\' here but changed to the `alias` directive instead of `root`

Reply:
Okay, I see.